bugwarrior/config.py

```diff
@@ -150,28 +155,36 @@ def validate_config(config, main_section):
         get_service(service).validate_config(config, target)
 
 
-def load_config(main_section, interactive=False):
-    config = ConfigParser({'log.level': "INFO", 'log.file': None})
+def get_config_path():
+    """
+    Determine the path to the config file. This will return, in this order of
+    precedence:
+    - the value of $BUGWARRIORRC if set
+    - $XDG_CONFIG_HOME/bugwarrior/bugwarriorc if exists
+    - <dir>/bugwarrior/bugwarriorc if exists, for dir in $XDG_CONFIG_DIRS
+    - ~/.bugwarriorrc if exists
+    - $XDG_CONFIG_HOME/bugwarrior/bugwarriorc otherwise
+    """
```

Comment:
I can add `BUGWARRIORC` but I also wanted to document the previous undocumented change (the xdg stuff), and now that I look closer at it, there are some things I'm not sure of. In particular:

* `~/.config/bugwarrior/bugwarriorrc` is now preferred over `~/.bugwarriorrc`. I don't think that's wrong but I want to make sure it's intentional.
* `/etc/xdg/bugwarrior/bugwarriorrc` will allo take precedence  over `~/.bugwarriorrc`. While it might make sense to have a system-wide bugwarrior configuration, I'd prefer if it was the other way around, i.e. that it would only be used if `~/.bugwarriorrc` is **not** present.

So I would like to at least change the order from

1. `$XDG_CONFIG_HOME`
2. `$XDG_CONIFG_DIRS`
3. `~/.bugwarriorrc`

to

1. `$XDG_CONFIG_HOME`
2. `~/.bugwarriorrc`
3. `$XDG_CONIFG_DIRS`

Any objection?